pkg/quarterdeck/tokens/tokens.go

```diff
@@ -218,6 +218,7 @@ func (tm *TokenManager) CreateRefreshToken(accessToken *jwt.Token) (refreshToken
 			NotBefore: jwt.NewNumericDate(accessClaims.ExpiresAt.Add(tm.conf.RefreshOverlap)),
 			ExpiresAt: jwt.NewNumericDate(accessClaims.IssuedAt.Add(tm.conf.RefreshDuration)),
 		},
+		OrgID: accessClaims.OrgID,
```

Comment:
This adds the `OrgID` to the refresh token claims so that they will be available during the `Refresh` process.